src/AWS.Deploy.Constants/CLI.cs

```diff
@@ -6,6 +6,8 @@ namespace AWS.Deploy.Constants
 {
     internal static class CLI
     {
+        public const string DEFAULT_AWS_REGION = "us-east-1";
```

Comment:
I don't want accidental use of this constant. Can you rename to `DEFAULT_STS_AWS_REGION` with a comment that this should only be used in this use case.

Reply:
Fixed